kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/dto.scala

```diff
@@ -69,3 +69,13 @@ case class GetFunctionsRequest(
     functionName: String)
 
 case class OpActionRequest(action: String)
+
+case class ResultSetMetaData(columns: Seq[ColumnDesc])
+
+case class ColumnDesc(
+    columnName: String,
+    comment: String,
+    position: Int,
+    dataType: String,
+    precision: Int,
+    scale: Int)
```

Comment:
let's reorder the params to columnName, dataType, columnIndex, precision, scale, comment

Reply:
ok